src/software/ai/navigator/navigator.h

```diff
@@ -150,4 +147,7 @@ class Navigator : public IntentVisitor
      * @return A factor from 0 to 1 for how close p is to an enemy obstacle
      */
     double getCloseToEnemyObstacleFactor(Point &p);
+
+   private:
+    std::unique_ptr<PathPlanner> path_planner_;
```

Comment:
uber nit: are we still doing _ at the end of private members? I thought this was a ROS convention

Reply:
This is inconsistent in the code base, for example tactic.h: done_. We should decide on a convention and file an issue to make it consistent

Reply:
Mat and I talked about this a while back, but basically said we didn't want to force a standard here if we couldn't automatically check it. However `clang-tidy` will do this for us. Could you file a ticket to decide/implement this and make it blocked on https://github.com/UBC-Thunderbots/Software/issues/839?

Reply:
#915